ilc/server/tailor/configs-injector.js

```diff
@@ -26,14 +26,9 @@ module.exports = class ConfigsInjector {
 
         let document = template.content;
 
-        if (
-            typeof document !== 'string' ||
-            !document.includes('<html') ||
-            !document.includes('</head>') ||
-            !document.includes('<head>') ||
-            !document.includes('</body>') ||
-            !document.includes('<body>')
-        ) {
+        const validHtmlRegex = /<html\s*.*?>.*<head\s*.*?>.*<\/head>.*<body\s*.*?>.*<\/body>.*<\/html>/s;
```

Comment:
duplicated regexp

Reply:
It's a different app. Do we have here some place for shared constants/utils between apps?